### PR TITLE
broken link fix to developers guide

### PR DESCRIPTION
**Pull request by raymondnijssen**
Changed this link to the right page. I am a but unsure about the new link containing the translation code in the path though...

### CODE REVIEW

File: source/site/getinvolved/development/qgisdevelopersguide/qtcreator.rst [hunk 1]
```diff
@@ -27,7 +27,8 @@ Setting up your project
 
 I'm assuming you have already got a local QGIS clone containing the
 source code, and have installed all needed build dependencies etc. There are
-`detailed instructions <http://github.com/qgis/QGIS/blob/master/CODING.md>`_.
+`detailed instructions 
+<http://qgis.org/en/site/getinvolved/development/qgisdevelopersguide.html>`_.
```
Comment (DelazJ) [inline source/site/getinvolved/development/qgisdevelopersguide/qtcreator.rst:31]: Thanks @raymondnijssen 
Actually this link was active at the time of QGIS 2.12 and no longer exists (or shouldn't). I suppose the right link to use would be http://qgis.org/en/site/getinvolved/development/qgisdevelopersguide/index.html or http://qgis.org/en/site/getinvolved/development/qgisdevelopersguide/codingstandards.html

To ensure translated link, you can use the [`:ref:` role](http://docs.qgis.org/2.14/en/docs/documentation_guidelines/writing.html#label-reference) i.e, :ref:`detailed instructions <QGIS-Developers-Guide>`. 
QGIS-Developers-Guide is an anchor text at the top of the source file of http://qgis.org/en/site/getinvolved/development/qgisdevelopersguide/index.html, so clicking the text will lead to that page, keeping the current language.

Comment (raymondnijssen) [reply source/site/getinvolved/development/qgisdevelopersguide/qtcreator.rst:31]: Hi @DelazJ , I got this link from @m-kuhn on the mailing list:

http://osgeo-org.1560.x6.nabble.com/broken-link-to-CODING-md-td5295647.html

That link still works...

Comment (raymondnijssen) [reply source/site/getinvolved/development/qgisdevelopersguide/qtcreator.rst:31]: I don't know what info the old CODING.md contained, but I expect something about dependencies for compiling. That is not in any of the urls mentioned now.

All that kind of information is (according to me and Google) still on the github pages.

Comment (m-kuhn) [reply source/site/getinvolved/development/qgisdevelopersguide/qtcreator.rst:31]: The CODING.md contained what is now on http://qgis.org/en/site/getinvolved/development/qgisdevelopersguide/codingstandards.html

**Comment by rduivenvoorde** [comment]
Thanks for fixing!

**Comment by m-kuhn** [comment]
@rduivenvoorde is there no need for a `:ref:`-link to have it properly translated?

**Comment by DelazJ** [comment]
done in 470a862

**Pull request merged**